installer/config/ui-asset-config.yaml

```
@@ -0,0 +1,57 @@
+    lambdaConfig:
+      handler: lambda_function.lambda_handler
+      runtime: python
+      runtimeVersion: latest
+      timeout: 10
+      isArm64: true
+      permissions:
+        requireS3: false
+        requireDynamoDB: true
+        requireSecretsManager: true
+      layers:
+        - name: svc-asset-vioaltions-layer
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)